ScreenToGif/Model/ApplicationViewModel.cs

```diff
@@ -878,16 +878,18 @@ internal void SendFeedback()
             }
         }
 
-        internal async void CheckForUpdates()
+        internal async Task CheckForUpdates(object obj)
```

Comment:
The reason why I use Task here has two reasons:

1. The function is async/await, we must call "await CheckUpdate(false);" to wait for the confirmation whether we need update or not.

2. We don't suggest using "void async..." except for an event, see: https://docs.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming#avoid-async-void

3. Add an object as an indicator for checking user's settings or not, because "Check For Update" has nothing to do with user's settings, it should update by just a click.